src/DBFTPlugin/PrepareRequest.cs

```diff
@@ -10,12 +10,14 @@ public class PrepareRequest : ConsensusMessage
         public uint Version;
         public UInt256 PrevHash;
         public ulong Timestamp;
+        public byte[] VRFProof;
         public UInt256[] TransactionHashes;
 
         public override int Size => base.Size
             + sizeof(uint)                      //Version
             + UInt256.Length                    //PrevHash
             + sizeof(ulong)                     //Timestamp
+            + VRFProof.GetVarSize()             // Nonce
```

Comment:
If it's fixed size (81) we should use 81
```suggestion
            + sizeof(ulong)                     //Timestamp
            + 81                                // Nonce
```